components/formats-gpl/src/loci/formats/in/FujiReader.java

```diff
@@ -142,17 +142,16 @@ public void close(boolean fileOnly) throws IOException {
   /* @see loci.formats.FormatReader#initFile(String) */
   @Override
   protected void initFile(String id) throws FormatException, IOException {
-    super.initFile(id);
-
-    if (checkSuffix(id, "inf")) {
-      infFile = new Location(id).getAbsolutePath();
-      pixelsFile = infFile.substring(0, infFile.lastIndexOf(".")) + ".img";
-    }
-    else {
+    if (!checkSuffix(id, "inf")) {
       pixelsFile = new Location(id).getAbsolutePath();
```

Comment:
Probably this line can be removed, since `pixelsFile` will be reset in line 154?

Reply:
:+1: Fixed in[f7230ae](https://github.com/ome/bioformats/pull/4267/commits/f7230aea92b6e28176899776e33a406bf1dfd8cd)